apps/site/src/components/layout/Header.astro

```diff
@@ -0,0 +1,103 @@
+---
+import { Icon } from "astro-icon/components";
+import Logo from "./Logo.astro";
+
+const links = [
+  {
+    href: "/",
+    text: "Home",
+  },
+  {
+    href: "/projects",
+    text: "Projects",
+  },
+  {
+    href: "/docs",
+    text: "Documentation",
+  },
+  {
+    href: "/blog",
+    text: "Articles",
+  },
+];
+
+const socialLinks = [
+  {
+    href: "/",
+    text: "Twitter (X)",
+    icon: "ri:twitter-x-fill",
+  },
+  {
+    href: "/docs",
+    text: "GitHub",
+    icon: "ri:github-fill",
+  },
+  {
+    href: "/blog",
+    text: "Discord",
+    icon: "ri:discord-fill",
+  },
+];
+---
+
+<header
+  class="bg-white sticky top-0 z-20 border-b border-b-gray-200 font-heading"
+>
```

Comment:
This will lead to two nested `header` tags, which is not optimal. We should keep the solution from before with the dynamic element tag. (If we have agreement, I'm happy to change it back if you're busy?)

```
---
const Element = Astro.props.as || 'header';
---

<Element>

</Element>
```